ext/mro/mro.xs

```diff
@@ -177,7 +177,7 @@ S_mro_get_linear_isa_c3(pTHX_ HV* stash, U32 level)
                     HE* const he = hv_fetch_ent(tails, seqitem, 1, 0);
```

Comment:
nope. lvalue fetch means that the result needs to be SV*, not NULL. The result needs to be assignable.